apps/omg_watcher/lib/omg_watcher/web/controllers/transaction.ex

```diff
@@ -63,4 +64,22 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
       |> api_response(conn, :create)
     end
   end
+
+  defp increment_metrics_counter(response) do
+    case response do
+      {:error, {:validation_error, _, _}} ->
+        Appsignal.increment_counter("transaction.failed.validation", 1)
+
+      {:error, {error, _}} when is_atom(error) ->
```

Comment:
@InoMurko What do you think about:
```suggestion
      {:error, {error, _}} when error == :client_error or error == :server_error or error == :malformed_response ->
```
or 
```suggestion
      {:error, {error, _}} when error == :client_error 
                           when error == :server_error 
                           when  error == :malformed_response ->
```
or maybe something else

Reply:
I'll repeat a comment I've made in a discussion we've had with Paweł - do we even want this counter? Mind this is counting whatever a bypass endpoint to a child chain is doing, so close to nothing.

Reply:
I will add these errors to 'transaction.fail.unidentified'